compiler+runtime/test/bash/main-exit/pass-test

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bb
+
+(ns jank-test-main-exit
+  (:require [babashka.process :as proc]
+            [clojure.string :as str]
+            [clojure.test :as t :refer [deftest run-test is]]))
+
+(def this-nsym (ns-name *ns*))
+
+(defn is-shell-exit [expected-code & cmd]
+  (let [{:keys [exit] :as res} (apply proc/shell {:out :string
+                                                  :err :string
+                                                  :continue true}
+                                      cmd)]
+    (is (= expected-code exit)
+        (pr-str (select-keys res [:exit :out :cmd])))
+    res))
+
+(deftest jank-exit-code-test
+  (is-shell-exit 0 "jank --module-path src run-main jank-test.successful-exit-code")
+  (is-shell-exit 1 "jank --module-path src run-main jank-test.fail-throw-exit")
+  (is-shell-exit 1 "jank --module-path src run-main jank-test.fail-uncatchable-exit")
+  (is-shell-exit 106 "jank")
```

Comment:
Why 106?

Reply:
I'm guessing it's from here https://github.com/jank-lang/CLI11/blob/5a03ee58384ce18b6b3de5a2256e18f5ef103633/include/CLI/Error.hpp#L52